### PR TITLE
Add documentation page for Multi-dataset Fitting

### PR DESCRIPTION
**Pull request by samueljackson92**
**Description of work.**
As part of #22766 I made some interface docs because none existed.

**To test:**

 - Build the docs.
 - Check them

_No associated issue_

Does this update require release notes?
- [ ] Yes
- [ ] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by martyngigg** [comment]
@samueljackson92 Should the target branch here be `release-next`?

**Comment by samueljackson92** [comment]
@martyngigg it is now.

File: docs/source/interfaces/Multi-dataset Fitting.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+After adding a function a new entry will appear in the top left panel with the
+name of the fit function and a list of fitting parameters. The user can enter
+appropriate guesses at starting values for the model parameters. Once a model
+has been created the "Plot Guess" checkbox can be used to show a plot of the
+model in the spectrum viewer using the starting parameter. To fit the model
+click the "Fit" button.
+
+
+.. figure::  ../images/multidataset-fitting/multidataset-fit-prop-dialog.png
+   :align: center
```
Comment (mantid-roman) [inline docs/source/interfaces/Multi-dataset Fitting.rst]: It is the same picture as above. Maybe it should be an image showing the newly added function?

File: docs/source/interfaces/Multi-dataset Fitting.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+---------------
+
+The spectrum viewer is shown in the bottom panel of the interface. This shows a
+plot of a single spectrum from the loaded list. It also shows a plot of the
+model with the current parameters overlaid on the spectrum. Using the buttons
+shown in the screenshot below the user can select which spectrum they wish to
+view. The plot is interactive and can be panned and zoomed. 
+
+The user can also interactively select `x` bounds either for a single spectrum
+or for all spectra by selecting the "Apply to all spectra option). To select
```
Comment (mantid-roman) [inline docs/source/interfaces/Multi-dataset Fitting.rst]: There is a closing bracket ) but no opening one.

File: docs/source/interfaces/Multi-dataset Fitting.rst [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+clicking on an entry.
+
+.. figure::  ../images/multidataset-fitting/multidataset-spectra.png
+   :align: center
+   :width: 800px
+
+   The list of spectra after loading a workspace.
+
+Fit Property Browser
+--------------------
```
Comment (mantid-roman) [inline docs/source/interfaces/Multi-dataset Fitting.rst]: It is important to mention how to set global parameters. There is also a dialog that helps setting ties on local parameters.

File: docs/source/interfaces/Multi-dataset Fitting.rst [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+split into two major parts. Th top panel allows the user to define the
+specifics of the model they wish to fit to the dataset. The bottom panel
+contains the parameters for the minimizer and general parameters not associated
+with the model (e.g. the number of iterations). The bottom panel also allows
+the user to choose between simultaneous and sequential fitting mode:
+
+- **Simultaneous Fitting**: Fit all spectra at the same time as a multi-domain
+  function.
+
+- **Sequential Fitting**: Fit the spectra one after another, using the
```
Comment (mantid-roman) [inline docs/source/interfaces/Multi-dataset Fitting.rst]: Sequential fitting has two modes set via `Fit Type` property:
1. Sequential, in which the fitted parameters of the previous spectrum are used as the starting values for the next one.
2. Individual, when all spectra have the same starting values.

**Review by mantid-roman — CHANGES_REQUESTED** [review]

File: docs/source/interfaces/Multi-dataset Fitting.rst (hunk 3 above)
Comment (samueljackson92) [reply docs/source/interfaces/Multi-dataset Fitting.rst]: I will also add a section on this.

File: docs/source/interfaces/Multi-dataset Fitting.rst (hunk 4 above)
Comment (samueljackson92) [reply docs/source/interfaces/Multi-dataset Fitting.rst]: Thanks! I didn't know we could operate in both modes.

**Review by mantid-roman — APPROVED** [review]

**Pull request merged**